src/index.js

```diff
@@ -327,22 +329,38 @@ class HCaptcha extends React.Component {
     }
 
     execute (opts = null) {
+
+      opts = typeof opts === 'object' && opts !== null ? opts : null;
```

Comment:
opts !== null check seems to be redundant
as opts are set to null anyway

`opts = typeof opts === 'object' ? opts : null;`